src/Runner.Worker/Container/ContainerHooks/HookResponse.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace GitHub.Runner.Worker.Container.ContainerHooks
+{
+    public abstract class HookResponse
+    {
+        public JToken State { get; set; }
+        public virtual void Validate() { }
+    }
+    public class PrepareJobResponse : HookResponse
```

Comment:
do we need the abstract class if we only have `PrepareJobResponse` that overrides it?

Reply:
Should have been `HookResponse` :/ Fixed it.